cftlib/nfdiv/src/main/java/uk/gov/hmcts/divorce/solicitor/event/SolicitorOrgDetails.java

```diff
@@ -0,0 +1,48 @@
+package uk.gov.hmcts.divorce.solicitor.event;
+
+import uk.gov.hmcts.divorce.divorcecase.model.UserRole;
+
+import java.util.Arrays;
+import java.util.Optional;
+
+public enum SolicitorOrgDetails {
+
+    CREATOR("ecb8fff1-e033-3846-b15e-c01ff10cb4bb", "10", UserRole.CREATOR.getRole()),
```

Comment:
Can we get rid of this enum and just have the account creation inline in the create test case event? otherwise it looks like a permanent part of the application here

Reply:
Not removing it as we need Adam ids and roles to create role assignments